packages/expo-av/ios/EXAV/Video/EXVideoManager.m

```diff
@@ -29,8 +29,7 @@ - (void)setModuleRegistry:(EXModuleRegistry *)moduleRegistry
 
 - (UIView *)view
 {
-  id<EXAVInterface> avModule = [_moduleRegistry getModuleImplementingProtocol:@protocol(EXAVInterface)];
-  return [[EXVideoView alloc] initWithAvModule:avModule];
+  return [[EXVideoView alloc] initWithModuleRegistry:_moduleRegistry];
```

Comment:
This is actually reverting what I made in #18633, but this view manager is unused anyway and I'm going to remove it from the repo soon.